.github/workflows/rust.yml

```diff
@@ -30,4 +30,15 @@ jobs:
       - name: Cargo build
         uses: actions-rs/cargo@v1
         with:
-          command: build 
+          command: check
```

Comment:
This command should stay as 'build' because 'check' does not try to link. As we use a custom link script, some link errors may be unnoticed if we only use 'check'